exporter/opentelemetry-exporter-prometheus/src/opentelemetry/exporter/prometheus/__init__.py

```diff
@@ -217,6 +218,7 @@ def _translate_to_prometheus(
                     metrics.append(metric)
 
         for metric in metrics:
+            label_keyss = []
```

Comment:
I would recommend using a more descriptive name such as `labels_keys_list` and `label_values_list`